packages/kit/src/module/install.ts

```diff
@@ -106,9 +108,17 @@ export async function loadNuxtModuleInstance (nuxtModule: string | NuxtModule, n
             url: nuxt.options.modulesDir.map(m => directoryToURL(m.replace(/\/node_modules\/?$/, '/'))),
             extensions: nuxt.options.extensions,
           })
-
-        nuxtModule = await jiti.import(src, { default: true }) as NuxtModule
         resolvedModulePath = fileURLToPath(new URL(src))
+        if (!existsSync(resolvedModulePath)) {
```

Comment:
i'm presuming we save some performance here by not passing in a non-existent file to `jiti.import`